sewer/dns_providers/common.py

```diff
@@ -73,5 +52,30 @@ def delete_dns_record(self, domain_name, domain_dns_value):
 
         This method should return None
         """
-        self.logger.info("delete_dns_record")
         raise NotImplementedError("delete_dns_record method must be implemented.")
+
+    def fulfill_authorization(self, identifier_auth, token, acme_keyauthorization):
+        """
+        https://tools.ietf.org/html/draft-ietf-acme-acme-18#section-8.4
+        A client fulfills this challenge by constructing a key authorization
+        from the "token" value provided in the challenge and the client's
+        account key.  The client then computes the SHA-256 digest [FIPS180-4]
+        of the key authorization.
+
+        The record provisioned to the DNS contains the base64url encoding of
+        this digest.  The client constructs the validation domain name by
+        prepending the label "_acme-challenge" to the domain name being
+        validated, then provisions a TXT record with the digest value under
+        that name.  For example, if the domain name being validated is
+        "example.org", then the client would provision the following DNS
+        record:
+        """
+        domain_name = identifier_auth["domain"]
+        base64_of_acme_keyauthorization = calculate_safe_base64(
+            sha256(acme_keyauthorization.encode("utf8")).digest()
+        )
+        self.create_dns_record(domain_name, base64_of_acme_keyauthorization)
+        return {"domain_name": domain_name, "value": base64_of_acme_keyauthorization}
+
+    def cleanup_authorization(self, domain_name, value):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameters differ from overridden 'cleanup_authorization' method](https://app.codacy.com/manual/komuW/sewer/pullRequest?prid=5008291)

Reply:
closing this in favor of discussion above about same issue